src/util/util.ts

```diff
@@ -482,16 +454,16 @@ export function isImageBitmap(image: any): image is ImageBitmap {
  * ArrayBuffers.
  *
  * @param data - Data to convert
- * @param callback - A callback executed after the conversion is finished. Invoked with error (if any) as the first argument and resulting image bitmap (when no error) as the second
+ * @returns - A  promise resolved when the conversion is finished. Invoked with error (if any) as the first argument and resulting image bitmap (when no error) as the second
```

Comment:
```suggestion
 * @returns - A  promise resolved when the conversion is finished
```

Sorry, I think I forgot to remove the now-irrelevant stuff in my suggestion